rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_storage.cpp

```diff
@@ -33,6 +33,19 @@
 #include "rosbag2_storage_default_plugins/sqlite/sqlite_statement_wrapper.hpp"
 #include "rosbag2_storage_default_plugins/sqlite/sqlite_exception.hpp"
 
+#ifdef _WIN32
```

Comment:
nitpick: i know i have also copied this - is there a simple way for us to dedupe this block?

Reply:
I feel like that's a thing which should be handled upstream. Not sure how to properly address this within the context of rosbag2.

Reply:
Ah, yeah, I see that we're using it in 3 different packages that each independently depend on `yaml_cpp_vendor` (`rosbag2_storage`, `rosbag2_transport`, and now `rosbag2_storage_default_plugins`). I might have suggested we have a common `rosbag2_cpp/yaml.hpp` that had this preprocessor patch, but rosbag2_cpp doesn't depend on yaml now.

Anyways, definitely not going to block anything on this. How big of a patch do you think upstream needs to fix this, do you have a sense of that?